tests_integration/__tests__/loglevel.js

```diff
@@ -40,11 +41,13 @@ function runPrettierWithLogLevel(logLevel, patterns) {
 
   expect(result).not.toEqual(0);
```

Comment:
drive-by comment: should probably not be negated at all to assert on the actual value instead of what it's _not_ to avoid false negatives such as this.

Reply:
I'm going to change it to

```js
expect(result.status).toEqual(2);
```